### DISCUSSION TITLE
目录和文件权限安全问题

### DISCUSSION THREAD

**Discussion by Dr-TSNG** [1Panel-dev/1Panel#6778] [Ideas]
### 1Panel 版本

v1.10.19-lts

### 请描述您的需求或者改进建议

现在 `/opt/1panel` 下大部分目录和文件都是所有人可读可执行，在本机存在多用户的情况下十分不安全。虽然将所有服务都容器化是一种缓解方案，但是把所有 supervisor 管理的小程序都容器化实在是过于浪费。

### 请描述你建议的实现方案

1. 创建文件的默认权限改为 640（至少不要可执行）
2. 不能让任何本机用户都能读取 `/opt/1panel` 下绝大部分文件
3. 使用 docker userns 功能，避免容器中的用户/组和本机用户/组重合导致权限泄露（比如本机 1000 用户和 php 容器用户重合了，导致可以任意访问修改网页目录）

**Comment by wanghe-fit2cloud** [comment]
感谢反馈，对于上述需求目前有些疑问。可以加我的企业微信号，方便详细讨论一下需求细节。
待具体需求产出后，我们将在后续版本中支持。​
<img width="223" alt="image" src="https://github.com/user-attachments/assets/e48b9175-241d-463c-ac3e-498d5d081abf">
